src/NzbDrone.Common/Disk/DiskProviderBase.cs

```diff
@@ -189,6 +189,17 @@ public long GetFileSize(string path)
             }
 
             var fi = new FileInfo(path);
+
+            // If the file is a symlink, resolve the target path and get the size of the target file.
+            if ((fi.Attributes & FileAttributes.ReparsePoint) == FileAttributes.ReparsePoint)
+            {
+                var targetPath = fi.ResolveLinkTarget(true)?.FullName;
+                if (targetPath != null)
```

Comment:
New line before if block

```suggestion

                if (targetPath != null)
```